### PR TITLE
Bump primer to 0.49.1

### PR DESCRIPTION
**Pull request by HDinger**

### CODE REVIEW

File: frontend/src/global_styles/primer/_overrides.sass [hunk 1]
```diff
@@ -114,10 +103,6 @@ sub-header,
     border-radius: 0px 0px var(--borderRadius-medium) var(--borderRadius-medium)
   .op-ckeditor--wrapper
     margin-bottom: 0px
-  
-// Todo: Remove once https://github.com/primer/view_components/pull/3087 is merged
-.FormControl-spacingWrapper
-  row-gap: var(--stack-gap-normal)
```
Comment (akabiru) [inline frontend/src/global_styles/primer/_overrides.sass:120]: ~Should we also clear other references? I see two direct references in `frontend/src/app/shared/components/autocompleter/draggable-autocomplete/draggable-autocomplete.component.html`~

Ah, nvm- incorrect deduction! 👍🏾

Comment (Kharonus) [inline frontend/src/global_styles/primer/_overrides.sass:120]: 🥳 yay

**Review by Kharonus — APPROVED** [review]
LGTM, 🟢 if @akabiru has no objections

**Review by akabiru — APPROVED** [review]
🥇

**Pull request merged**